Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:openjdk-8-jre-alpine
+FROM eclipse-temurin:8-alpine
```

Comment:
```suggestion
# TODO: Bump to JDK11 and generate a JRE with jlink - https://hub.docker.com/_/eclipse-temurin#CreatingaJREusingjlink
FROM eclipse-temurin:8-jre-alpine
```

- The image is aimed at running in a production environment so no JDK at all. 
- Add a note for later JDK11 user to generate a JRE with jlink (like what we do on jenkinsci/docker)